datacube/api/core.py

```diff
@@ -6,14 +6,14 @@
 import uuid
 import collections.abc
 from itertools import groupby
-from typing import Set, Union, Optional, Dict, Tuple, cast
```

Comment:
Type hinting is supported by newer collections : https://docs.python.org/3.9/whatsnew/3.9.html#type-hinting-generics-in-standard-collections

Reply:
Yes, that's why I'm removing the old typehint-only ones.